ee/debug/shipper/README.MD

```diff
@@ -1,20 +1,31 @@
-shipping initiated by user (cmd line):
+## Flare Upload Flow
+
+The shipper supports uploading flares to both **GCS (Google Cloud Storage)** and **S3 (Amazon S3)** using presigned/signed URLs.
+
+### Shipping initiated by user (cmd line):
 ```mermaid
 sequenceDiagram
-    launcher->>+server: signed url request
-    server->>+gcp: create signed url
-    gcp-->>+server: signed url
-    server-->>+launcher: signed url
+    launcher->>+server: presigned url request
+    server->>+cloud: create presigned url (GCS or S3)
```

Comment:
Nitpick -- maybe instead of "cloud" we call this "flare storage"? Same for the other diagram